ci/builders/nodejs-lambda-codebuild.yaml

```diff
@@ -0,0 +1,157 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: CodeBuild for maven docker project
+Parameters:
+  GitHubProjectName:
+    Description: Name of pagopa project name
+    Type: String
+  BaseDir:
+    Description: base directory of the lambda project
+    Type: String
+  CiArtifactBucket:
+    Description: Ci Artifact bucket name
+    Type: String
+  NotificationSNSTopic:
+    Type: String
+    Description: Topic for build and pipeline notification
+
+Resources:
+  CodeBuildProject:
+    Type: AWS::CodeBuild::Project
+    Properties:
+      Name: !Sub "${GitHubProjectName}-${BaseDir}-CodeBuild"
+      ServiceRole: !GetAtt CodeBuildServiceRole.Arn
+      Environment:
+        PrivilegedMode: true
```

Comment:
Sicuramente serve per docker. Ma serve anche in questo caso ?

Reply:
eseguito test funzionante settando PrivilegedMode: false